app/src/main/res/xml/appfilter.xml

```diff
@@ -924,9 +939,6 @@
     <!-- Google Clock -->
     <item component="ComponentInfo{com.google.android.deskclock/com.android.deskclock.DeskClock}" drawable="clock" />
 
-    <!-- Google Contacts -->
-    <item component="ComponentInfo{com.google.android.contacts/com.android.contacts.activities.PeopleActivity}" drawable="people" />
```

Comment:
same here

Reply:
As mentioned in the commit message of e8cc47163f05af14680574fce02010637c482d04

This line references a drawable `people` that doesn't actually exist and is most likely the root cause for dkanada/frost#246
At least from my understanding this definition would overwrite the previous 2 (now 1) lines 465 (now 478) and 471 (now removed).

Reply:
Ah I see now, you are indeed correct!